build/config/constants.js

```diff
@@ -25,14 +22,15 @@ const release_config = {
     production: {
         branch        : 'master',
         target_folder : '',
-        valid_sections: ['app', 'app_2', 'all'],
+        valid_sections: ['app_2'],
         origin        : release_targets.production.repo,
+        target_repo   : release_targets.production.target_repo,
         CNAME         : release_targets.production.CNAME,
     },
     staging: {
         branch        : 'master',
         target_folder : '',
-        valid_sections: ['app', 'app_2', 'all'],
+        valid_sections: ['app_2'],
```

Comment:
same, is `section` necessary?

Reply:
we need too many changes to remove it, we should handle that in #6 or the release process card.